workflow/hosts/hera.yaml

```
@@ -17,3 +17,4 @@ chgrp_cmd: 'chgrp rstprod'
 hpssarch: 'YES'
 localarch: 'NO'
 atardir: '/NCEPDEV/$HPSS_PROJECT/1year/$USER/$machine/scratch/$PSLOT'
+supported_resolutions: ['C384', 'C192', 'C96', 'C48']
```

```suggestion
supported_resolutions: ['C768', 'C384', 'C192', 'C96', 'C48']
```
Presentation from Jongil on 9/21 clarified that the coupled model was run at C768 atmosphere without modifications in the workflow with the exception of manually staging initial conditions.